protocol/consistencychecks.go

```diff
@@ -118,9 +118,33 @@ func (cc *ConsistencyChecks) updateSTR(requestType int, msg *Response) error {
 		}
 		// Otherwise, expect that we've entered a new epoch
 		if err := cc.verifySTRConsistency(cc.SavedSTR, str); err != nil {
+			// FIXME: we are returning the error immediately
+			// without saving the inconsistent STR
+			// see: https://github.com/coniks-sys/coniks-go/pull/74#commitcomment-19804686
```

Comment:
You mean because the registration and key lookup don't fetch the new STR as we discussed in #173 ?